ShellPkg/Application/Shell/ShellProtocol.c

```diff
@@ -3251,7 +3251,7 @@ EfiShellSetCurDir (
       return (EFI_INVALID_PARAMETER);
     }
 
-    //    gShellCurMapping = MapListItem;
+    gShellCurMapping = MapListItem;

```

Comment:
Please explain in the commit log how this change is related to the problem you are addressing.